corehq/apps/app_manager/tests/test_advanced_suite.py

```diff
@@ -22,27 +24,28 @@
 from corehq.util.test_utils import flag_enabled
 
 
+@patch('corehq.apps.app_manager.suite_xml.post_process.resources.get_xform_overrides', return_value=[])
```

Comment:
Its not clear to me why this needs a mock at all. That function should return an empty dict when called if a record doesn't exist. Also the function you're patching returns dicts, but all of these patches return empty lists, which I believe is unintentional

Reply:
The empty list bit is unintentional, I'll fix that. The patches are to avoid the db call so these tests don't have to switch from `SimpleTestCase` to `TestCase`.

Reply:
Ah. avoiding testcase does make sense. This is used enough times and is easy enough to mess up that you may want to consider creating a utility in app manager's test utils. We do something similar for UCRs https://github.com/dimagi/commcare-hq/blob/fd9f7aa24f25093683e17a69bb4a14f44d0e15b7/corehq/apps/userreports/tests/utils.py#L115-L117

Reply:
Wrapped the patch in a function and fixed the return value in 842d51abb84

I did not remove many of the `*args` that I added earlier, and am not planning to unless there's a compelling reason.